### PR TITLE
Update fastlane-plugin-revenuecat_internal and fix release-train job

### PR DESCRIPTION
**Pull request by vegaro**
https://github.com/RevenueCat/purchases-ios/pull/1862#issuecomment-1226835780

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -623,9 +620,10 @@ workflows:
...
       - danger
   weekly-run-workflow:
-    when:
-      and:
-        - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
-        - equal: [ "release-train", << pipeline.schedule.name >> ]
+    # when:
+      # and:
+        # - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+        # - equal: [ "release-train", << pipeline.schedule.name >> ]
```
Comment (taquitos) [inline .circleci/config.yml:626]: Probably unintentional to leave this in, right?

**Review by taquitos — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (vegaro) [reply .circleci/config.yml:626]: oh yes, I just commented this to trigger it the job lol, I wanna see if it works as expected

Comment (vegaro) [reply .circleci/config.yml:626]: I actually think there's a way of running this pipeline from CircleCI. Next time. The job is almost done I don't want to cancel it

Comment (vegaro) [reply .circleci/config.yml:626]: actually, let's merge it and run it on main

**Comment by vegaro** [comment]
Going to merge without `ci/circleci: run-test-ios-16` since it's unrelated

**Pull request merged**